nav2_theta_star_planner/test/test_theta_star.cpp

```diff
@@ -0,0 +1,176 @@
+//  Copyright 2020 Anshumaan Singh
```

Comment:
We can wait on CI to see - but I would like this to be at 90%+ coverage to be in line with the new coverage % in Nav2 (since you submitted this PR, our average went up to 87% so this needs to at least meet that)

Reply:
It should be so, i have covered as much functions as possible